openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/README.md

```diff
@@ -0,0 +1,69 @@
+# MedMNIST 2D Classification Using FedProx Optimizer Tutorial
+
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
+
```

Comment:
More information about FedProx / a link to the paper should also be included here

Reply:
I made this change.